utils/annotations.py

```diff
@@ -145,7 +140,7 @@ def set_annotation_class(self, annotation):
         if annotation["line"]["color"] in map_color_to_class:
             self.annotation_class = map_color_to_class[annotation["line"]["color"]]
         else:
-            self.annotation_class = 99
+            self.annotation_class = 99  # todo: change this method to work with @tammy's #57 PR - new color picker features
```

Comment:
since map_color_to_class won't work anymore due to the dynamic annotations feature, we need to wait for #57 where a feature that stores colors and class names of the annotations will be added.
We will then take class names for every annotation, and add it appropriately. 

Until then all classes will be placeholder value 99